src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts

```diff
@@ -80,17 +81,44 @@ Ensure that you are able to run Docker without using sudo directly with the
 sudo usermod -aG docker $USER
 exec newgrp docker
 \`\`\`
-3. Install the dockstore command-line program and add it to the path.
+
+#### Part 1 for MacOS - Install dependencies
+1. We'll cover two ways to install Java 11. One way is to download the JDK from [Oracle](https://www.oracle.com/java/technologies/javase/jdk11-archive-downloads.html). Choose the MacOS Compressed Archive Version and follow the commands below.
```

Comment:
For consistency, this should also use OpenJDK (like our existing tests, we will likely need to write Mac tests that use OpenJDK for licensing/philosophical reasons)

Reply:
Using link to openJDK instead